package.json

```diff
@@ -33,6 +33,7 @@
     "axios": "^0.21.1",
```

Comment:
under scripts,
 "build:lib": "rm -rf ./js ./sgds/sgds.css && webpack --mode=production",

 since there is no more ./js folder generated in webpack anymore

 "build:lib": "rm -rf  ./sgds/sgds.css && webpack --mode=production"